karma/makeconf.js

```diff
@@ -71,6 +121,19 @@ function makeConf(defaultFile, browserNoActivityTimeout, requires) {
           base: 'Electron',
           flags: ['--default-user-agent']
         },
+        FirefoxInDocker: {
+          base: 'Firefox',
+          flags: [
+            '-headless',
+          ],
+          prefs: {
+            'media.gstreamer.enabled': false,
+            'media.navigator.permission.disabled': true,
+            'media.navigator.streams.fake': true,
+            'media.autoplay.enabled.user-gestures-needed': false,
+            'media.block-autoplay-until-in-foreground': false
```

Comment:
You may want to consider setting following 2 values as well:
'media.getusermedia.insecure.enabled'
'media.devices.insecure.enabled'

Future builds of FF does not support connecting to insecure URLs, I assume that including localhost.

Reply:
Thanks, I will experiment with those and add them if they work well.